rasa/core/agent.py

```diff
@@ -289,11 +289,11 @@ def create_agent(model: Text, endpoints: Text = None) -> "Agent":
     from rasa.core.tracker_store import TrackerStore
     from rasa.core.utils import AvailableEndpoints
     from rasa.core.brokers.broker import EventBroker
-    import rasa.utils.common
+    import asyncio
 
     _endpoints = AvailableEndpoints.read_endpoints(endpoints)
 
-    _broker = rasa.utils.common.run_in_loop(EventBroker.create(_endpoints.event_broker))
+    _broker = asyncio.run(EventBroker.create(_endpoints.event_broker))
```

Comment:
Will this actually work? Pretty sure we already have a running event loop at this point so `asyncio.get_event_loop` seems to be more fitting, right?

Reply:
Tests seem to pass so it does work but it might be more efficient to use the helper to run it in the existing loop if that exists, I might do an experiment to see if doing that actually takes the `get_event_loop` path

Reply:
that would be great cause I'm pretty sure there should already be an existing loop